screens/dashboard/index.js

```diff
@@ -204,71 +204,71 @@ class Dashboard extends Component<PropsType> {
                         showExpandButton={(incompleteGoals || []).length > 1}
                         title={'CURRENT GOALS:'}
                     >
                         <GoalMessageBox
                             goal={incompleteGoals[0]}
                             message={currentGoalVerbiage}
-                            gotoDetails={() => navigation.navigate('GoalDetails', {goal: incompleteGoals[0] || {}})}
+                            gotoDetails={() => navigation.navigate('GoalDetails', { goal: incompleteGoals[0] || {} })}
                             updateGoal={updateGoal(incompleteGoals[0])}
                         />
                         {this.state.expanded1 && allButFirst(incompleteGoals)}
                     </GoalsBox>
                     <GoalsBox
                         onExpand={isExpanded => {
-                            this.setState({expanded2: isExpanded});
+                            this.setState({ expanded2: isExpanded });
                         }}
                         showExpandButton={(submittedGoals || []).length > 1}
                         title={'SUBMITTED GOALS:'}
                     >
                         <GoalMessageBox
                             goal={submittedGoals[0]}
                             message={firstSubmittedGoalVerbiage}
-                            gotoDetails={() => navigation.navigate('GoalDetails', {goal: submittedGoals[0] || {}})}
+                            gotoDetails={() => navigation.navigate('GoalDetails', { goal: submittedGoals[0] || {} })}
                             updateGoal={updateGoal(submittedGoals[0])}
                         />
                         {this.state.expanded2 && allButFirst(submittedGoals)}
                     </GoalsBox>
                     <GoalsBox
                         onExpand={isExpanded => {
-                            this.setState({expanded3: isExpanded});
+                            this.setState({ expanded3: isExpanded });
                         }}
                         showExpandButton={(completedGoals || []).length > 1}
                         title={'COMPLETED GOALS:'}
                     >
                         <GoalMessageBox
                             goal={completedGoals[0]}
-                            gotoDetails={() => navigation.navigate('GoalDetails', {goal: completedGoals[0] || {}})}
+                            gotoDetails={() => navigation.navigate('GoalDetails', { goal: completedGoals[0] || {} })}
                             message={firstCompletedGoalVerbiage}
                             updateGoal={() => (void 0)}
                         />
                         {this.state.expanded3 && allButFirst(completedGoals)}
 
                     </GoalsBox>
-                    <Text style={{textAlign: 'center', lineHeight: 30}}>Version {pkg.version}</Text>
+                    <Text style={{ textAlign: 'center', lineHeight: 30, color: '#427CAB' }}>Version {pkg.version}</Text>
```

Comment:
Sweet, thanks Paul, that looks great. 

I assume this is the only substantive change in this PR.  Let me know if there were others. 🙏

Reply:
Yep!  That's it.   Thanks, and happy to help.